projects/projectRoutes.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express');
+const knex = require('knex');
+ 
+const dbConfig = require('../knexfile');
+ 
+const dbProjects = knex(dbConfig.development);
+
+const router = express.Router();
+
+router.get('/', (req,res) => {
+    dbProjects('projects')
+    .then (projects =>{
+    res.status(200).json(projects)
+    })
+    .catch(err => res.status(500).json(err))
+});
+
+router.get('/:id', (req,res) => {
+    const {id} = req.params;
+    dbProjects('projects')
+    .where({ id })
+    .first()
+    .then(project => {
+        if(project) {
+            dbProjects('actions')
+            .where({ project_id: id })
+            .then(actions => {
+                project.actions = actions;
+                 res.status(200).json(project);
+            })
+            .catch(err => res.json(err))
+        } else {
+            res.status(404).json({ error: 'Unable to locate project.'})
+        }
+    })
+    .catch(err => res.status(500).json(err));
+})
+
+router.post('/', (req, res) => {
+    const project = req.body;
```

Comment:
Sanitize your data coming in instead of taking the entire request body. Specify which properties make up a project and take only those from the request.